### DISCUSSION TITLE
How can a QMD file see R code present in another QMD file?

### DISCUSSION THREAD

**Discussion by piiskop** [quarto-dev/quarto-web#182] [Q&A]
In _bookdown_, i can use variables and functions written in R defined in one RMD-file in another RMD-file. How can i do the same with _quarto_?

**Comment by jjallaire** [comment]
In Quarto each chapter is rendered in a separate session (this is so that you can work on a book a single chapter at a time rather than having to render the entire book each time). You can still manage to share data/objects though via explicit use of the R  `save()` and `load()` functions.
